crates/block-producer/src/block_builder/prover/tests.rs

```diff
@@ -373,11 +371,11 @@ async fn compute_account_root_empty_batches() {
     // Set up account states
     // ---------------------------------------------------------------------------------------------
     let account_ids = [
-        AccountId::new_unchecked(Felt::new(0b0000_0000_0000_0000u64)),
-        AccountId::new_unchecked(Felt::new(0b1111_0000_0000_0000u64)),
-        AccountId::new_unchecked(Felt::new(0b1111_1111_0000_0000u64)),
-        AccountId::new_unchecked(Felt::new(0b1111_1111_1111_0000u64)),
-        AccountId::new_unchecked(Felt::new(0b1111_1111_1111_1111u64)),
+        AccountId::new_unchecked([Felt::new(0b0000_0000_0000_0000u64), Felt::ZERO]),
+        AccountId::new_unchecked([Felt::new(0b1000_0000_0000_0000u64), Felt::ZERO]),
+        AccountId::new_unchecked([Felt::new(0b1100_0000_0000_0000u64), Felt::ZERO]),
+        AccountId::new_unchecked([Felt::new(0b1110_0000_0000_0000u64), Felt::ZERO]),
+        AccountId::new_unchecked([Felt::new(0b1111_0000_0000_0000u64), Felt::ZERO]),
```

Comment:
Similarly to the comment above, the first felt contains metadata that can be invalid if we just fill it with ones (the `new_unchecked` method does some small validations despite its name).